tests/smokescreen/test_compartment_5.py

```diff
@@ -151,28 +156,24 @@ def test_compartment_5(page: Page, smokescreen_properties: dict) -> None:
     # Select 'Record other post-investigation contact' button
     page.get_by_role("button", name="Record other post-").click()
 
-    # Complete 'Contact Direction',   To patient
-    # 'Contact made between patient and',  Selects the top option in the dropdown
+    # Complete 'Contact Direction', To patient
+    # 'Contact made between patient and', Selects the top option in the dropdown
     # 'Date of Patient Contact',  Today
     # 'Duration',  01:00
     # 'Start Time',  11:00
     # 'End Time',  12:00
     # 'Discussion Record'   TEST AUTOMATION
     # select 'Outcome' - 'Post-investigation Appointment Not Required' and click 'Save'
-    page.locator("#UI_DIRECTION").select_option(label="To patient")
-    page.locator("#UI_CALLER_ID").select_option(index=0)
-    page.get_by_role("button", name="Calendar").click()
-    CalendarPicker(page).v1_calender_picker(datetime.today())
-    page.locator("#UI_START_TIME").click()
-    page.locator("#UI_START_TIME").fill("11:00")
-    page.locator("#UI_END_TIME").click()
-    page.locator("#UI_END_TIME").fill("12:00")
-    page.locator("#UI_COMMENT_ID").click()
-    page.locator("#UI_COMMENT_ID").fill("Test Automation")
-    page.locator("#UI_OUTCOME").select_option(
-        label="Post-investigation Appointment Not Required"
-    )
-    page.get_by_role("button", name="Save").click()
+
+    ContactWithPatientPage(page).select_direction_dropdown_option("To patient")
+    ContactWithPatientPage(page).select_callerid_dropdown_option(index=0)
```

Comment:
Instead of `index = 0,` just have `1`
```suggestion
    ContactWithPatientPage(page).select_callerid_dropdown_option(1)
```